packages/contracts/deploy/30_plugins/10_plugin-repos/00_create-plugins-repo.ts

```diff
@@ -10,29 +15,61 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     'Please make sure pluginRepo is not created more than once with the same name.'
   );
 
+  const {network} = hre;
+
   // AddresslistVotingSetup
+  const addresslistCIDPath = await uploadToIPFS(
+    JSON.stringify(addresslistMetadata),
+    network.name
+  );
   await createPluginRepo(
     hre,
     'AddresslistVoting',
     'AddresslistVotingSetup',
     [1, 0, 0],
-    '0x'
+    ethers.utils.hexlify(
+      ethers.utils.toUtf8Bytes(`ipfs://${addresslistCIDPath}`)
```

Comment:
probably, @mathewmeconry can better review this part until i remember ipfs

Reply:
amended